src/ui.cpp

```diff
@@ -533,21 +534,16 @@ UpdateDialog::UpdateDialog()
 
     m_buttonSizer = new wxBoxSizer(wxHORIZONTAL);
 
+    const wxString REMIND_OPTIONS[] = { L"Download now", L"Remind me in an hour", L"Remind me tomorrow", L"Remind me in a week" };
```

Comment:
Wondering if we still want all of these remind options. Seems fine to me but UX hasn't probably reviewed this in a while.

Reply:
We might improve on this later (as part of another ticket). I did not want any UX involvement at this point to avoid any delays getting the underlying plumbing work done first.